docs/application/dotnet/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,607 @@
+# Component Based Applications
```

Comment:
TOC needs to be updated.

Reply:
I fixed it.
Please recheck them.
Thank you :)